core/src/test/java/kafka/test/junit/RaftClusterInvocationContext.java

```diff
@@ -77,7 +77,7 @@ public String getDisplayName(int invocationIndex) {
             (BeforeTestExecutionCallback) context -> {
                 KafkaClusterTestKit.Builder builder = new KafkaClusterTestKit.Builder(
                     new TestKitNodes.Builder().
-                        setNumKip500BrokerNodes(clusterConfig.numBrokers()).
+                            setNumBrokerNodes(clusterConfig.numBrokers()).
```

Comment:
indentation seems a little weird here

Reply:
Not sure, it must have auto-formatted when i renamed to `setNumBrokerNodes`